osu.Game/Screens/Menu/MainMenu.cs

```diff
@@ -86,11 +87,8 @@ private void load(OsuGame game = null)
                 }
             };
 
-            if (game != null)
-            {
-                buttons.OnSettings = game.ToggleSettings;
```

Comment:
Broke because it's cached as `MainSettings` now (`Cache` vs `CacheAs<SettingsOverlay>`).